sdk/docs/iot/README.md

```diff
@@ -93,47 +72,58 @@ Feature | Azure SDK for Embedded C | Description
 
 ## Examples
 
+These examples introduce you to the API calls for a few key features of the Embedded C SDK.
+
+- [IoT Hub Client Initialization](#iot-hub-client-initialization)
+- [Properties](#properties)
+- [Telemetry](#telemetry)
```

Comment:
We should make these more descriptive (what do they do), like `IoT Hub Client Initialization` and update the corresponding headings that these link to, too.

Reply:
@jspaith or @danewalton any thoughts on this?

Reply:
i don't have strong opinions but if you add it ideally it should be on brief side since the actual gist is in the link itself.